loony_dev/git.py

```
@@ -62,6 +66,41 @@ def force_commit_and_push(self, message: str) -> None:
+            cwd=self.work_dir,
+            capture_output=True,
+            text=True,
+        )
+        if commit_proc.returncode != 0:
+            output = f"{commit_proc.stdout}\n{commit_proc.stderr}".strip()
+            logger.debug("git commit failed: %s", output)
+            if any(kw in output.lower() for kw in _HOOK_KEYWORDS):
+                raise HookFailureError(output)
+            raise GitError(output)
+
+        push_proc = subprocess.run(
```

Ruff (0.15.11)

[error] `subprocess` call: check for execution of untrusted input

(S603)